### PR TITLE
feat(combobox): add tokens and update shim

### PR DESCRIPTION
**Pull request by Jinnie**
## PR Checklist

Please check if your PR fulfills the following requirements:

- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)
- [ ] If applicable, have a visual design approval

## PR Type

What kind of change does this PR introduce?

- [ ] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [ ] Documentation content changes
- [ ] Other... Please describe:

## What is the current behavior?

Issue Number: N/A

## Does this PR introduce a breaking change?

- [ ] Yes
- [ ] No

### CODE REVIEW

**Review by mindarelus — APPROVED** [review]
lgtm

File: projects/angular/src/popover/dropdown/_menu-mixins.clarity.scss [hunk 1]
```diff
@@ -140,10 +145,21 @@
...
   }
+
+  // From typography SECONDARY-13RG-exp == P5
+  @include mixins.css-var(font-size, clr-dropdown-font-size, inherit, variables.$clr-use-custom-properties);
+  @include mixins.css-var(letter-spacing, clr-dropdown-letter-spacing, inherit, variables.$clr-use-custom-properties);
+  @include mixins.css-var(line-height, clr-dropdown-line-height, inherit, variables.$clr-use-custom-properties);
 }
 
 @mixin generate-dropdown-item-height() {
   padding: variables.$clr_baselineRem_0_125 variables.$clr_baselineRem_1;
```
Comment (dtsanevmw) [inline projects/angular/src/popover/dropdown/_menu-mixins.clarity.scss:156]: We probably don't need that with the mixin fallback below/

**Review by dtsanevmw — COMMENTED** [review]
Overall looks good, just couple of questions / suggestions to consider.

- The icons in the pills looks like that there are going more to the bottom. Is that something we are okey with or?
Screenshot 2023-08-13 at 22 08 00

- I see that the trigger (caret) icon is 16x16 in the spec but they didn't add anything specific for that

- Not something which we need to change in this PR but I see that we already have the trigger ( caret ) button inline in the combobox template but we still have some styles for `position: absolute` which don't look necessary and we can remove them and all the paddings would fit perfectly.  With these styles removed it won't need the gap mixin and it can be a lot cleaner.

File: projects/angular/src/popover/dropdown/_menu-mixins.clarity.scss (hunk 1 above)
Comment (Jinnie) [reply projects/angular/src/popover/dropdown/_menu-mixins.clarity.scss:156]: Yes, thanks! Forgot to remove it.

**Review by dtsanevmw — APPROVED** [review]
Looks good!

**Pull request merged**